pkg/scheduler/scheduler.go

```diff
@@ -466,7 +466,9 @@ func (sched *Scheduler) scheduleOne() {
 					" No preemption is performed.")
 			} else {
 				preemptionStartTime := time.Now()
-				sched.preempt(pod, fitError)
+				if err := sched.preempt(pod, fitError); err != nil {
```

Comment:
How about keeping return (node, err) same as before, and printing it if err == nil? It might be useful for debugging.